packages/account/src/Sections/Security/LoginHistory/__test__/login-history.spec.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { useLoginHistory, APIProvider } from '@deriv/api';
+import { isMobile } from '@deriv/shared';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import LoginHistory from '../login-history';
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Loading: jest.fn(() => 'mockedLoading'),
+}));
+
+jest.mock('@deriv/api', () => ({
+    ...jest.requireActual('@deriv/api'),
+    useLoginHistory: jest.fn(),
+}));
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(() => false),
+}));
+
+const mockUseLoginHistory = useLoginHistory as jest.MockedFunction<typeof useLoginHistory>;
+
+type TReturnUseLoginHistory = {
+    data: {
+        formatted_data: [{ [key: string]: string | number }];
+    };
+    isLoading: boolean;
+    isError: boolean;
+    error: unknown;
+};
+
+describe('<LoginHistory />', () => {
+    let mock_store: ReturnType<typeof mockStore>, response: TReturnUseLoginHistory;
```

Comment:
```suggestion
    let mock_store: ReturnType<typeof mockStore>, response: ReturnType<typeof useLoginHistory>;
```
can you please check if it works? if so, we can remove TReturnUseLoginHistory

Reply:
I've tried using this to type it, but it forces me to include all the returning values which are unnecessary into the response that i've declared 😓 Is there any other way to type it 🙏

Reply:
Partial<ReturnType<typeof useLoginHistory>>
can you try this?

Reply:
Tried this also couldn't work 😅